_episodes/03-expertise.md

```diff
@@ -120,9 +133,10 @@ seem trivial to you, as they often will.
 > This discussion should take about 5 minutes.  
 {: .challenge}
 
+Another potential challenge for experts who teach is what is called *expert blind spot*.
 Experts are frequently so familiar with their subject
 that they can no longer imagine what it's like to *not* see the world that way.
-This is called *expert blind spot* and can lead to what's known as the *expertise-reversal effect* - experts are often
+This is called expert blind spot and can lead to what's known as the *expertise-reversal effect* - experts are often
```

Comment:
why remove the italics here? At some point we may need to consider whether "blind spot" in this instance is ableist language. However as it is the prevailing term and generally well-understood by learners, I think it needs to stay for now.

Reply:
It now occurs earlier and I italicized it there. But that's kind of awkward. Let me fix.

Reply:
I'm not 100% happy with this, but don't know how to change it. :(